android/java/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/annotations/MarkerOptions.java

```diff
@@ -100,7 +105,7 @@ public MarkerOptions snippet(String snippet) {
 
     public MarkerOptions sprite(@Nullable String sprite) {
         if (!TextUtils.isEmpty(sprite)) {
-            ((Marker)annotation).sprite = sprite;
+            ((Marker)annotation).setSprite(sprite);
```

Comment:
This will cause breakages with JNI as described in https://github.com/mapbox/mapbox-gl-native/issues/2551.